consensus/core/src/tx.rs

```diff
@@ -154,9 +168,7 @@ impl Transaction {
             gas,
             payload,
             id: Default::default(), // Temp init before the finalize below
```

Comment:
The comment `// Temp init before the finalize below` is obsolete

Reply:
Fixed in [aef51ab](https://github.com/kaspanet/rusty-kaspa/pull/371/commits/aef51aba5ce1ff40d3c3d38f8d83fbb6318e4508)